focus/doc/bazel_tutorial.md

```diff
@@ -0,0 +1,28 @@
+# Using focus to build bazel
+If you want to test-out focus, but don't have a monorepo to setup and test against, we recommend trying focus against the github.com/bazelbuild/bazel repo.
+
+## Setup Steps
+First you will need to follow the directions in the top-level README.md to install focus.
+
+Then you will need to install `brew install openjdk@11`.
+
+## Clone bazelbuild/bazel
+If focus is not already in your path, you may want to temporarily add it with:
+
+`export PATH="/Users/dbernadett/workspace/focus/target/debug:$PATH"`
+
+You can then `focus new` a new sparse repo:
+
+`focus new --dense-repo=https://github.com/bazelbuild/bazel --template bazel bazel-focus`
```

Comment:
Use a triple-quoted code block for this kind of tutorial step (can't suggest it on Github because the code suggestion also uses triple backquotes).